GenesToPhenotypes.ipynb

```diff
@@ -1709,13 +1729,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 26,
+   "execution_count": 17,
    "metadata": {},
    "outputs": [],
    "source": [
     "conn.close()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Exercises\n",
+    "\n",
+    "To run the exercises below, you will need to be connected to IDR. If you have run the cell in the notebook closing the connection to IDR i.e. ``conn.close()``, you will first need to re-run the cell connection to IDR or alternatively add a new cell using ``Insert > Cell below``, add ``conn = connection('idr.openmicroscopy.org')`` and run the cell.\n",
```

Comment:
Could a newline be added after "add" word ? Otherwise the ``conn = `` ends up on another line than the rest of the command, which led me to copying just the command from the equal sign on, and crash.
```